apps/daimo-mobile/src/view/screen/send/SendTransferButton.tsx

```diff
@@ -119,14 +124,34 @@ export function SendTransferButton({
   })();
   const disabled = sendDisabledReason != null || dollars === 0;
 
+  let handlePress = undefined;
+  if (landlineBankTransferOption === BankTransferOptions.Deposit) {
+    assert(
+      recipient.type === "landlineBankAccount",
+      "Recipient of Landline deposit must be a landline bank account"
+    );
+    // TODO: authenticate this call
+    handlePress = () => {
+      const rpcFunc = getRpcFunc(daimoChainFromId(account.homeChainId));
+      rpcFunc.depositFromLandline.mutate({
+        daimoAddress: account.address,
+        landlineAccountId: recipient.landlineAccountId,
+        amount: dollarsStr,
+        memo,
+      });
```

Comment:
wait, we're on the send transfer screen. aren't we sending to a Landline receiving address here? (= withdraw to bank account)

Reply:
ah, nvm, i see